README.md

```diff
@@ -52,3 +52,36 @@ Running Tests
 - To run the simple http request tests, first click the ResponseTestSuite link, then click the Suite button.
 - To run the tests that require threading, first click the SimultaniousTestSuite link, then click the Suite button.
 - To run a test individually, first click on the test link, then click the Test button.
+
+Adding CI
+------------
+- Fork cob spec
+- On your local machine navigate to your server's directory
+- Create a submodule using the forked copy of cob spec `git submodule add https://github.com/myusername/cob_spec`
```

Comment:
Should this be the SSH version of the URL in case someone wants to push changes to their fork?